process/tests/fuser/mod.rs

```diff
@@ -86,16 +78,31 @@ fn get_process_user(pid: u32) -> io::Result<String> {
         .parse()
         .map_err(|_| io::Error::new(io::ErrorKind::InvalidData, "Invalid UID"))?;
 
+    get_username_by_uid(uid)
+}
+
+#[cfg(target_os = "macos")]
+fn get_process_user(pid: u32) -> io::Result<String> {
+    use std::os::unix::fs::MetadataExt;
+
+    let uid = fs::metadata("/").map(|md| md.uid())?;
```

Comment:
Yes, I changed it with `libc::getuid()`, but I am not sure that is a good option. Maybe that is enough for testing because, for a better one, I need to re-implement the `stat()` function in tests from `fuser.rs`.